extensions/interactions/TextInput/protractor.js

```diff
@@ -30,9 +30,15 @@ var customizeInteraction = async function(elem, placeholderText, heightOfBox) {
 var expectInteractionDetailsToMatch = async function(
     elem, placeholderText, heightOfBox) {
   if (placeholderText) {
-    expect((await element(by.tagName('oppia-interactive-text-input'))
-      .getAttribute('placeholder-with-value')))
-      .toEqual('&quot;' + placeholderText + '&quot;');
+    placeholderValue = (
+      await element(by.tagName('oppia-interactive-text-input'))
+        .getAttribute('placeholder-with-value')
```

Comment:
Please make sure this element has loaded (using `waitFor`) before interacting with it

Reply:
I don't believe my code changes the original behavior here.. do I still need waitFor here? @U8NWXD

Reply:
In general, yes, please make these kinds of simple fixes even if the old code wasn't written well. To draw an analogy to a more obvious problem, if we had code with obvious stylistic issues, e.g. 

```js
for(i=1;i<10;i=i+1){arr[i].click()}
```

we wouldn't just leave it, right? We'd fix it even if we didn't introduce the problem. In this case though, I'm willing to make an exception since this PR is such a big migration.

Reply:
@shavavo could you please handle this comment in the follow up PR?

Reply:
Sure- thanks @U8NWXD